generate/proto.go

```diff
@@ -45,6 +46,11 @@ func protoStringOrNil(s string) *string {
 	return proto.String(s)
 }
 
+func protoEnumValue(value constant.Value) int32 {
+	val, _ := constant.Int64Val(value)
```

Comment:
I'd prefer if we do `if !ok { panic(...) }`. Otherwise, we might hide bugs or invalid input since we happily accept those as zero values.

Talking about hypotheticals of course. But I prefer to ensure assumptions via panics, since that means programs will break in less subtle ways if something goes wrong.

Reply:
My understanding was that this would get picked up in the `constant.Int64Val` function, it panics if the value is not an int, and since the gunk tags are type checked there couldn't be a situation where the type is anything but an int? I have added the panic for safe keeping regardless.

Reply:
Oh, the `!ok` happens if the value isn't exact, not if it's not an integer. A value can be non-exact if it's unknown because the program had a type error somewhere. So I do think we should check, just in case.

Reply:
Ah, gotcha! Updated wording.